students/Briadean/lesson02/grid_printer.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+"""Print an exact replica of the example grid"""
+def grid_printer_1():
+    row = "+" + " - " * 4 + "+" + " - " * 4 + "+"
```

Comment:
Nbd, but you have a space on either side of your dashes (" - ") which makes your squares wider than they were supposed to be. (This puts two spaces between the middle dashes, and one between the dash and the plus.)